guidelines/enhancement_template.md

```diff
@@ -115,6 +117,18 @@ bogged down.
 
 Include a story on how this proposal will be operationalized:  lifecycled, monitored and remediated at scale.
 
+### API Extensions
+
+- name the API extensions (CRDs, admission and conversion webhooks, aggregated API servers) 
+  this enhancement adds or modifies
+- does this enhancement modify the behaviour of existing resources, especially those owned 
+  by other parties than the authoring team (including upstream resources), and if yes how
+
+  Examples:
```

Comment:
Do you want to add a specific example of something owned by someone else and how you'd like to see that described?

Reply:
the examples below are like that. A finalizer on namespaces modifies a kube object. A label restriction modifies another object normal behaviour.

Reply:
Q: if it's owned by someone else do you want them to explicitly say it like you requested like in failure modes below? Should they be added as reviewers?

Reply:
I would leave that to the API review to judge. Some APIs have owners that can be added. If the modification is more generic, i.e. to multiple resources, it's not that clear.

Reply:
lgtm I just don't have perms to resolve my comment :)